docx/oxml/numbering.py

```diff
@@ -259,12 +277,51 @@ def get_start_override(for_numId):
         p_num = count_same_numIds(preceding_paragraphs_numIds, numId, p_num)
 
         try:
+            # apply numbering style
             p_num = self.fmt_map[lvl_el.numFmt.get('{%s}val' % nsmap['w'])](p_num)
         except KeyError:
             return None
 
+        suffix = ''
+        if append_suffix is True:
+            suffix = lvl_el.suffix
         lvlText = lvl_el.lvlText.get('{%s}val' % nsmap['w'])
-        return re.sub(r'%(\d)', str(p_num), lvlText, 1) + lvl_el.suffix
+        if lvlText.count('%') > 1:
+            # `lvlText` is an custom defined list label that has multiple numbering values
+            # e.g. `1.1.2`, `1.1.3`
+            prev_p_tuple = get_preceding_paragraph(p, ilvl)
+            if prev_p_tuple is not None:
```

Comment:
What would you say if we would reduce nesting here by inverting if condition and returning early? I find it easier to read and understand the code that way.

Reply:
Yes, you are right. I'm trying to include more early returns in my coding style, so this is a good example.